src/main/java/jenkins/plugins/openstack/compute/internal/Openstack.java

```diff
@@ -655,7 +655,7 @@ public void destroyFip(String fip) {
                             floatingIPv6 = address;
                         }
                     }
-                } else  if (Objects.equals(type, "fixed")) {
+                } else  if (Objects.equals(type, "fixed") || type == null) {
```

Comment:
I guess we do not need the `else if` here and should just accept everything.

Reply:
Done. I kept the existing tests as they might spots potential regressions.